pegen/c_generator.py

```diff
@@ -155,6 +156,13 @@ def visit_Repeat1(self, node: Repeat1) -> Tuple[str, str]:
         self.cache[node] = f"{name}_var", f"{name}_rule(p)"  # But not here!
         return self.cache[node]
 
+    def visit_Gather(self, node: Gather) -> Tuple[str, str]:
+        if node in self.cache:
+            return self.cache[node]
+        name = self.gen.name_gather(node)
+        self.cache[node] = f"{name}_var", f"{name}_rule(p)"  # No trailing comma here either!
```

Comment:
I'd just drop the comment here and on line 156 above; it no longer feels funny, and in C it's not about a trailing comma but about a comma operator.